### PR TITLE
Packs/Reco: update description and readme

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Reco/pack_metadata.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "Reco",
-    "description": "Reco - detects and protects against sensitive data leakage",
+    "description": "Reco is an identity-first SaaS security solution that empowers organizations with full visibility into every app, identity, and their actions to seamlessly prioritize and control risks in the SaaS ecosystem",
     "support": "partner",
-    "currentVersion": "1.2.0",
+    "currentVersion": "1.2.1",
```
Comment (yucohen) [inline Packs/Reco/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.2.0",
```

Comment (yucohen) [reply Packs/Reco/pack_metadata.json:5]: No need to bump version when changing README files

**Review by yucohen — COMMENTED** [review]
@GalNakash-RecoLabs Thanks for your contribution!

**Comment by GalNakash-RecoLabs** [comment]
Done

**Review by yucohen — APPROVED** [review]

**Pull request merged**